test/repositories/slick/implementations/ChatsRepositorySpec.scala

```diff
@@ -526,4 +526,33 @@ class ChatsRepositorySpec extends AsyncWordSpec with MustMatchers with BeforeAnd
 
   }
 
+  "SlickChatsRepository#moveChatToTrash" should {
+    val chatsRep = new SlickChatsRepository(db)
+
+    val userId = "148a3b1b-8326-466d-8c27-1bd09b8378f3" //beatriz@mail.com
+
+    "remove the user's chat from inbox, sent and draft and move it to trash" in {
+      val validChatId = "303c2b72-304e-4bac-84d7-385acb64a616"
+      for {
+        result <- chatsRep.moveChatToTrash(validChatId, userId)
+        optionUserChat <- db.run(UserChatsTable.all.filter(uc => uc.chatId === validChatId && uc.userId === userId).result.headOption)
+        userChat = optionUserChat.get
```

Comment:
Don't use get. Use the "Inside" trait from scala test.

Reply:
I didn't know that trait existed. I'm going to learn it right away :) Thanks!

Reply:
Wouldn't the OptionValues trait be more appropriate?